hibernate-core/src/main/java/org/hibernate/bytecode/enhance/spi/interceptor/EnhancementHelper.java

```diff
@@ -39,7 +46,7 @@ public static boolean includeInBaseFetchGroup(
 			if ( value instanceof ToOne ) {
 				if ( ( (ToOne) value ).isUnwrapProxy() ) {
```

Comment:
Due to the change below to `AnnotationBinder#defineFetchingStrategy`, if `@LazyToOne` is not mapped, then the following will be set:
```
toOne.setUnwrapProxy( true );
toOne.setUnwrapProxyImplicit( true );
```
and the message will end up getting DEBUG logged.

I think it should only get logged if NO_PROXY is explicitly mapped:

```
final ToOne toOne = (ToOne) value;
if ( toOne.isUnwrapProxy() && !toOne.isUnwrapProxyImplicit() ) {
```